src/System.Linq.Parallel/tests/QueryOperators/AggregateTests.cs

```diff
@@ -304,6 +304,17 @@ public static void Aggregate_OperationCanceledException(Labeled<ParallelQuery<in
         }
 
         [Theory]
+        [MemberData(nameof(UnorderedSources.Ranges), new[] { 128 }, MemberType = typeof(UnorderedSources))]
```

Comment:
I'll take a look at it.  I think I did get a little worried after running into the Concat problem.

Hmm, in Aggreagate's case, it's using either a custom enumerator that checks for the cancellation, or relies on the underlying enumerator (when it can't parallelize the reduce).  Probably I can pass the responsibility for the enumerator's cancellation to those tests, then just test with an single source here (and probably elsewhere, too, will check).  

Do you want me to reduce to fewer/single sources everywhere in this PR (where possible), or do should that be a new one?